department-of-veterans-affairs/vets-api
# API-38535-enable-hardcoded-wsdl-org-web-service

## Summary

- Hard codes OrgWebService. Adds a test file for find_definition.

## Related issue(s)

- [API-38634](https://jira.devops.va.gov/browse/API-38634)

## Testing done

- To test in Postman
1. call `{{uri}}/services/claims/v1/forms/2122`
2. place break point [here](https://github.com/department-of-veterans-affairs/vets-api/blob/API-38535-enable-hardcoded-wsdl-org-web-service/modules/claims_api/lib/bgs_service/local_bgs.rb#L320)
3. look at the endpoint variable, and it should be `PersonWebServiceBean/PersonWebService` or `ClaimantServiceBean/ClaimantWebService` 
4. determine if the definition is found


## What areas of the site does it impact?
	modified:   modules/claims_api/app/clients/claims_api/bgs_client/definitions.rb
	modified:   modules/claims_api/lib/bgs_service/local_bgs.rb
	modified:   modules/claims_api/spec/lib/claims_api/find_definition_spec.rb

## Acceptance criteria
